packages/client/hmi-client/src/services/goLLM.ts

```diff
@@ -0,0 +1,53 @@
+import API from '@/api/api';
+import useAuthStore from '@/stores/auth';
+import { TaskStatus } from '@/types/Types';
+import { logger } from '@/utils/logger';
+import { EventSourceMessage, fetchEventSource } from '@microsoft/fetch-event-source';
+
+export async function modelCard(documentId: string, modelId: string): Promise<any> {
+	const response = await API.post('/gollm/model_card', null, {
+		params: {
+			'document-id': documentId,
+			'model-id': modelId
+		}
+	});
```

Comment:
What is with the `null` parameter in your POST request?

Reply:
I thought that represented the body for the post request but its not needed, Ive removed it

Reply:
Oh nevermind, it is needed...ya the 2nd parameter represents the body for POST, but there is no body